### PR TITLE
#3744 revise Worker APIs document

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3744)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docs/src/rest/rest-workers.rst [hunk 1]
```diff
@@ -94,106 +94,46 @@ Example Request
...
+      "routes": {},
+      "config.storage.topic": "b8dadc3de21048fa927335b8f",
+      "sharedJarKeys": [],
+      "lastModified": 1578982566359,
       "tags": {},
-      "jarInfos": [],
-      "offset.storage.topic": "6ec3cd5f1b",
-      "group.id": "979a8c0c18",
+      "xmx": 2048,
+      "imageName": "oharastream/connect-worker:0.9.0-SNAPSHOT",
```
Comment (jackyoh) [inline docs/src/rest/rest-workers.rst]: $|version|

Comment (chia7712) [reply docs/src/rest/rest-workers.rst]: 喔 我又忘了 感謝提醒

Comment (jackyoh) [reply docs/src/rest/rest-workers.rst]: 是是是是

File: docs/src/rest/rest-workers.rst [hunk 2]
```diff
@@ -94,106 +94,46 @@ Example Request
...
-      "offset.storage.partitions": 1,
-      "status.storage.topic": "statusTopic",
-      "status.storage.replication.factor": 1,
-      "status.storage.partitions": 1,
-      "nodeNames": [
-        "node00"
-      ]
+      "name": "wk",
+      "nodeNames": ["node00"],
+      "brokerClusterKey": "bk"
```
Comment (vitojeng) [inline docs/src/rest/rest-workers.rst]: 這個 brokerClusterKey 的值是不是有錯?

Comment (chia7712) [reply docs/src/rest/rest-workers.rst]: 我們支援三種key的寫法

1.
```json
{
"brokerClusterKey": {
  "group": "default",
  "name": "n"
}
}
```

2.
```json
{
"brokerClusterKey": {
  "name": "n"
}
}
```

3.
```json
{
  "brokerClusterKey": "n"
}
```

**Pull request merged**